modules/biobank/php/collection.class.inc

```diff
@@ -0,0 +1,106 @@
+<?php declare(strict_types=1);
+/**
+ * Specimen Object Class
+ * This class contains an instance of a database handling that is needed to
+ * permanently store and retrieve Specimen Object instances.
+ *
+ * PHP Version 7.2
+ *
+ * @category   Entity
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\biobank;
+
+/**
+ * Specimen Object Class
+ * This class contains an instance of a database handling that is needed to
+ * permanently store and retrieve Specimen Object instances.
+ *
+ * PHP Version 7.2
+ *
+ * @category   Entity
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class Collection extends Process
+{
+    /**
+     * Persistent Instance variables.
+     *
+     * These variables are directly mapped to the fields of the database schema
+     * that describe a specimen entity.
+     *
+     * @access private
+     */
+    private $quantity; //float
+    private $unitId;   //int
+
+    /**
+     * Sets the current amount of specimen available
+     *
+     * @param float $quantity the current amount of specimen available
+     *
+     * @return void
+     */
+    public function setQuantity(float $quantity) : void
+    {
+        $this->quantity = $quantity;
+    }
+
+    /**
+     * Gets the current amount of specimen available
+     *
+     * @return float
+     */
+    public function getQuantity() : float
+    {
+        return $this->quantity;
+    }
+
+    /**
+     * Sets the ID of the specimen's unit of quantity
+     *
+     * @param string $unitId the ID of the quantity's unit
+     *
+     * @return void
+     */
+    public function setUnitId(int $unitId) : void
+    {
+        $this->unitId = $unitId;
+    }
+
+    /**
+     * Gets the ID of the specimen's unit of quantity
+     *
+     * @return string
+     */
+    public function getUnitId() : int
+    {
+        return $this->unitId;
+    }
+
+    /**
+     * Returns an array representing the state of the Specimen.
```

Comment:
That makes sense, I guess this will come into discussion as we started standardizing `DataInstance`/`Entity` classes, so I may hold off for now since I wouldn't be using that function anywhere in my PR.
@ridz1208 any thoughts?

Reply:
its not needed but it's pretty simple to do... I dont have any strong opinions

Reply:
Okay, we'll bring it up at our design meetings.